third_party/amd/lib/TritonAMDGPUToLLVM/ConvertLayoutOpToLLVM/SharedToDotOperandMFMA.cpp

```diff
@@ -198,13 +198,21 @@ Value convertLayout(int opIdx, ConversionPatternRewriter &rewriter,
                     const SharedMemoryObject &smemObj,
                     const LLVMTypeConverter *typeConverter, Value thread) {
   assert((opIdx == 0 || opIdx == 1) && "unexpected operand idx");
-  auto aTensorTy = cast<MemDescType>(tensor.getType());
+  auto tensorTy = cast<MemDescType>(tensor.getType());
+  auto aTensorTy = getExpandedDesc(tensorTy);
```

Comment:
What does "a" means in the name? It looks like we are getting the operand A here. Could you rename it?